### ISSUE TITLE
Add pre-built binaries to https://github.com/crc-org/crc/releases

### ISSUE THREAD

**Issue by cfergeau** [crc-org/crc#3413]
We are slowly moving away from our tray, we'll need to offer an alternative way to get our binaries at some point.
Even without taking into account the tray, having 'community' builds would be useful. In my opinion they make sense to have on https://github.com/crc-org/crc/releases but we could arrange something else.

We could upload the binaries generated from our rpm, but they are not standalone because of some `version.IsInstaller()` checks preventing them to automatically download the binaries they need, see https://github.com/crc-org/crc/blob/641ff18f521344a35df9e48f82fe31a49e2b20bd/pkg/crc/preflight/preflight_checks_darwin.go#L52-L67 for example.

Another option would be to build through github actions.

**Comment by anjannath** [comment]
Later we could also focus on packaging it for `homebrew`, `chocolatey` and `rpms`, `debs` this'd provide better install/upgrade experience

**Comment by cfergeau** [comment]
Yeah, homebrew availability would definitely be great to have. For macOS, we can do both, github.com/crc-org/crc/releases and homebrew
